code/iaas/ha/src/main/java/io/cattle/platform/ha/monitor/dao/impl/PingInstancesMonitorDaoImpl.java

```diff
@@ -28,9 +26,9 @@
                     .on(INSTANCE.ID.eq(INSTANCE_HOST_MAP.INSTANCE_ID))
                 .where(INSTANCE.STATE.eq(InstanceConstants.STATE_RUNNING)
                         .and(AGENT.ID.eq(agentId)))
```

Comment:
I don't think so. For the context in which this is called, we really just want all running instances associated with an agent, regardless of the agent's state.